### PR TITLE
Set up for my Portfolio Website and the first part of its mobile version

### PR DESCRIPTION
**Pull request by dd-obua**
## In this project I have:
* Set up linters to ensure code correctness
* Marked up the header and headline sections of the portfolio website
* Included CSS to style the markup according to specifications

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  <link rel="stylesheet" href="./css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800;900&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <header class="toolbar">
+    <img src="./img/menu-icon.svg" alt="Hamburger menu icon" class="hamburger-icon">
```
Comment (Ibizugbe) [inline index.html]: Nice job so far, your project looks great!👍. Let's just attend to this little issue. For semantic reasons, kindly wrap the hamburger bar in a `nav` tag😉.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @dd-obua 👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅
- [x] Linters are passing👍.
- [x] Github flow is used👍.
- [x] Descriptive commit messages👍.
- [x] `flexbox` is used to layout headline and toolbar section👍.
- [x] Project matches the design👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbeeraTahir — APPROVED** [review]
# Status: Approved ✔️✔️✔️  

Hi @dd-obua, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved

## Highlights
✔️ Nice implementation of the change requested by the previous reviewer 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dd-obua** [comment]
Thank so much for your approval.

**Pull request merged**